src/resolvers/auth.js

```diff
@@ -14,11 +14,16 @@ const {
 } = require('../middleware/auth');
 const { DASHBOARD_ROOT_PATH } = require('../constants/client-routes');
 
-const createResetPasswordEmail = resetToken => {
+const RESET_PASSWORD_TEMPLATE_ID = 'd-b444d727384e40f38efad1175b6681e7';
```

Comment:
Maybe extract these template IDs to `.env`?

Reply:
maybe will do in the future